src/containers/main/SideBar/components/Wallet/Wallet.tsx

```diff
@@ -40,17 +40,17 @@ export default function Wallet() {
         }
         if (transactions.length || is_reward_history_loading) {
             // Question(A): Not sure what this was for
-            setRecapCount(undefined);
+            setRewardCount(undefined);
```

Comment:
We could clear the reward count after opening the history. It looks like it's cleared too early